Website/src/views/HomeView.jsx

```diff
@@ -36,11 +36,11 @@ const HomeView = () => {
       <HomeHeader />
       <div className="main">
         {/* Live News Feeds/Updates (Te Ao, Te Hiku Media, Covid 19 - MOH & Iwi Leaders) */}
-        <NewsFeedSection />
```

Comment:
What's the reason for these changes? Appears unnecessary so please restore.

Looks like a formatting tool has done this; if so, please turn-off or don't-use in this project.